requirements.txt

```diff
@@ -5,3 +5,4 @@ cython>=0.23
 coverage>=4.0
 six>=1.10
 wheel>=0.29
+lxml>=3.6.0
```

Comment:
Sanity should be enough. Note that this Dockerfile is a template itself =)

https://github.com/grpc/grpc/blob/master/templates/tools/dockerfile/test/sanity/Dockerfile.template